### PR TITLE
[4.0] RavenDB-11317

### PR DESCRIPTION
**Pull request by ppekrol**
- logs should be in utc by default
- added the option to log in local server time zone

### CODE REVIEW

File: src/Raven.Server/Config/Categories/LogsConfiguration.cs [hunk 1]
```diff
@@ -14,5 +14,9 @@ public class LogsConfiguration : ConfigurationCategory
         [DefaultValue(LogMode.Operations)]
         [ConfigurationEntry("Logs.Mode", ConfigurationEntryScope.ServerWideOnly)]
         public LogMode Mode { get; set; }
+
+        [DefaultValue(true)]
```
Comment (ppekrol) [inline src/Raven.Server/Config/Categories/LogsConfiguration.cs:18]: @ayende not sure about that default. Your call.

File: src/Sparrow/Logging/Logger.cs [hunk 2]
```diff
@@ -13,6 +13,13 @@ public class Logger
         [ThreadStatic]
         private static LogEntry _logEntry;
 
+        private static readonly long LocalToUtcOffsetInTicks;
+
+        static Logger()
+        {
+            LocalToUtcOffsetInTicks = TimeZoneInfo.Local.GetUtcOffset(DateTime.Now).Ticks;
```
Comment (ayende) [inline src/Sparrow/Logging/Logger.cs:20]: Might need to update it every now and then for things like daylight savings changes?

Comment (ayende) [reply src/Sparrow/Logging/Logger.cs:20]: Can probably get away with doing that only once every file, or when we are about to go idle?

**Review by ayende — APPROVED** [review]

File: src/Sparrow/Logging/Logger.cs (hunk 2 above)
Comment (ayende) [reply src/Sparrow/Logging/Logger.cs:20]: I'll fix that in a bit

**Pull request merged**